plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java

```diff
@@ -190,11 +190,7 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @description
         // Returns a boolean indicating whether the specified location is inside this area.
         // -->
-        processor.registerTag(ElementTag.class, "contains", (attribute, area) -> {
-            if (!attribute.hasParam()) {
-                return null;
-            }
-            LocationTag loc = attribute.paramAsType(LocationTag.class);
+        processor.registerTag(ElementTag.class, LocationTag.class, "contains", (attribute, area, loc) -> {
             if (loc == null) {

```

Comment:
Can remove these `null` checks, as the new argument handling makes sure the param is present and valid

Reply:
Oh awesome! I'll push the changes right now :)